raw_description/urdf/arm_camera/arm_camera.urdf.xacro

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find cob_description)/urdf/common.xacro"/>
+  <xacro:include filename="$(find raw_description)/urdf/arm_camera/arm_camera.gazebo.xacro" />
+
+  <xacro:property name="arm_camera_length" value="0.043"/>
+  <xacro:property name="arm_camera_width" value="0.170"/>
+  <xacro:property name="arm_camera_height" value="0.126"/>
+  <xacro:property name="safe_value" value="0.01"/>
+
+  <xacro:macro name="arm_camera" params="name parent *origin">
+
+    <joint name="${name}_base_joint" type="fixed">
+        <xacro:insert_block name="origin"/>
+        <parent link="${parent}"/>
+        <child link="${name}_frame"/>
+    </joint>
+
+    <link name="${name}_frame"/>
+
+    <!-- joint between gripper_coupling_link and arm_camera_link -->
+    <joint name="${name}_joint" type="fixed">
+        <parent link="${name}_frame"/>
+        <child link="${name}_link"/>
+        <origin xyz=" 0.00 0.0 0.0" rpy="0 0.0 0.0"/>
+    </joint>
+
+    <link name="${name}_link">
+       <inertial>
+       <!--choose randomly-->
+        <origin xyz="0 0 0" rpy="0.0 0 0.0"/>
+        <mass value="0.0057307"/>
+        <inertia ixx="0.819" ixy="0" ixz="0" iyy="0.819" iyz="0" izz="0.819" />
+      </inertial>
+
+        <visual>
+            <origin xyz="0.083 0 0.0" rpy="0.0 0 0"/>
```

Comment:
this `origin` should be all zero...change the position via modifying the `origin` in the joint